src/components/Modal/Modal.scss

```diff
@@ -45,6 +45,7 @@
     width: 100%;
     max-height: 47.5rem;
     background-color: #fff;
+    margin-top: 2%;
```

Comment:
Margin-top should not be a percentage, because it will reference the element's available space. It should be a fixed value